File: CommonUtilities/GridViewCustomSort.cs
//****************************************************
//Modification History
//****************************************************
//Modified by          Date          Description
//****************************************************
//Manthan Rajguru      2015.09.16    YRS-AT-2550: YRS data cleanup: copyright comments and namespace (Added 'YMCARET' Namespace)
//*****************************************************
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
namespace YMCARET.CommonUtilities
{
    [Serializable()]
    public class GridViewCustomSort
    {

        string strSortExpression;
        string strSortDirection;
        public string SortExpression
        {
            get { return strSortExpression; }
            set { strSortExpression = value; }
        }

        /// <summary>
        /// 
        /// </summary>
        /// <value></value>
        /// <returns></returns>
        /// <exception cref="Test"/>
        /// <remarks></remarks>
        public string SortDirection
        {
            get { return strSortDirection; }
            set { strSortDirection = value; }
        }
    }
}

File: YMCAUI/CommonClasses/GridViewCustomSort.vb
'*******************************************************************************
' Copyright YMCA Retirement Fund All Rights Reserved. 
'
' Project Name		:	YMCA-YRS
' FileName			:	HelperFunctions.vb
' Author Name		:	Anudeep Adusumilli    
' Employee ID		:	56556
' Creation Time		:	17-december-2013
'*******************************************************************************

'************************************************************************************
'Modified By        Date            Description
'*********************************************************************************************************************
'Anudeep            25.03.2014      BT:957: YRS 5.0-1484 -Termination watcher changes Re-Work (YRS 5.0-1484)
'*********************************************************************************************************************
'AA:25.03.2014 - Added to not get error for termination watcher while storing object in viewstate
<Serializable()>
Public Class GridViewCustomSort

    Dim strSortExpression As String
    Dim strSortDirection As String
    Public Property SortExpression As String
        Get
            Return strSortExpression
        End Get
        Set(ByVal value As String)
            strSortExpression = value
        End Set
    End Property

    ''' <summary>
    ''' 
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <exception cref="Test"/>
    ''' <remarks></remarks>
    Public Property SortDirection As String
        Get
            Return strSortDirection
        End Get
        Set(ByVal value As String)
            strSortDirection = value
        End Set
    End Property
End Class
